remoting/etcdv3/client_test.go

```diff
@@ -295,13 +299,26 @@ func (suite *ClientTestSuite) TestClientWatch() {
 			t.Fatal(err)
 		}
 
+		events := make([]*mvccpb.Event, 0)
+
 		for e := range wc {
 
 			for _, event := range e.Events {
+				events = append(events, (*mvccpb.Event)(event))
 				t.Logf("type IsCreate %v k %s v %s", event.IsCreate(), event.Kv.Key, event.Kv.Value)
 			}
 		}
 
+		e1 := events[0]
+		e2 := events[1]
+
+		if e1.Type != mvccpb.PUT || string(e1.Kv.Key) != "name" || string(e1.Kv.Value) != "scott.wang" {
```

Comment:
```suggestion
		assert.Contains()
```
if array is not always sorting, suggest use assert.Contains()

Reply:
thanks for the hint, but for the sake of event queue semantic, the watching channel should be a sequential generator which means the event created in the first place should always be the first one consumed outside the watching channel IIRC, maybe I'm wrong.